cluster/get-kube-binaries.sh

```diff
@@ -143,13 +143,34 @@ function sha1sum_file() {
   fi
 }
 
+# Get default service account credentials of the VM.
+GCE_METADATA_INTERNAL="http://metadata.google.internal/computeMetadata/v1/instance"
+function get-credentials {
+  curl "${GCE_METADATA_INTERNAL}/service-accounts/default/token" -H "Metadata-Flavor: Google" -s | python -c \
+    'import sys; import json; print(json.loads(sys.stdin.read())["access_token"])'
+}
+
+function valid-storage-scope {
+  curl "${GCE_METADATA_INTERNAL}/service-accounts/default/scopes" -H "Metadata-Flavor: Google" -s | grep -q "auth/devstorage"
+}
+
 function download_tarball() {
   local -r download_path="$1"
   local -r file="$2"
+  local trace_on="off"
```

Comment:
@mikedanese do you think this will cover it?

Reply:
What does this do?